### ISSUE TITLE
Bug: Missing textures for abilities

### ISSUE THREAD

**Issue by 0928NPO031** [EladNLG/HUDRevampv2#3]
when i load up some important icons just missing
![image](https://github.com/EladNLG/HUDRevampv2/assets/141247817/8943e702-8a39-4f37-b5b2-1aeb6cb89ff2)
![image](https://github.com/EladNLG/HUDRevampv2/assets/141247817/f1f1eb4e-53ea-4213-8555-892cbb4ccb4d)
![image](https://github.com/EladNLG/HUDRevampv2/assets/141247817/bf9f40f8-e5ff-4508-b042-5bf0405ac6a8)
all of this is with just the hud pls help

**Comment by headassbtw** [comment]
yeah it do that

**Comment by TheEssketitMan** [comment]
What's weird is this only happens with things related to Monarch and Tone (from what I can tell so far.)
Hope it gets fixed soon!

**Comment by Gallerine** [comment]
yeah still getting this early 2024, hope this isn't a fully abandoned thing, I like the design but it feels half unplayable with Monarch

**Comment by EladNLG** [comment]
IRL stuff happens. Might find time to fix this, might not. Requires manual work of exporting icons and converting to VTFs.
